pkg/math_utils/utils_test.go

```diff
@@ -0,0 +1,34 @@
+package math_utils_test
+
+import (
+	. "github.com/lambdaclass/cairo-vm.go/pkg/math_utils"
+	"math/big"
+	"testing"
+)
+
+func div_mod_ok(t *testing.T) {
```

Comment:
The tests are not being executed, in order for Go to execute them you must write Test in the functions name, for example `TestDivModOk`